crates/ansi-to-html/benches/convert.rs

```diff
@@ -12,14 +13,52 @@ fn main() {
 
 static COMPRESSED_TERMINAL_SESSION: &[u8] = include_bytes!("../assets/terminal_session.gz");
 
-#[bench]
-fn convert(bencher: Bencher) {
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+enum Input {
+    AnsiHeavy,
+    PlainText,
+}
+
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+enum MaybeEsc {
+    Esc,
+    SkipEsc,
+}
+
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+enum MaybeOpt {
+    Opt,
+    SkipOpt,
+}
+
+#[bench(args = [
+    (Input::AnsiHeavy, MaybeEsc::Esc, MaybeOpt::Opt),
+    (Input::AnsiHeavy, MaybeEsc::Esc, MaybeOpt::SkipOpt),
+    (Input::AnsiHeavy, MaybeEsc::SkipEsc, MaybeOpt::Opt),
+    (Input::AnsiHeavy, MaybeEsc::SkipEsc, MaybeOpt::SkipOpt),
+    (Input::PlainText, MaybeEsc::Esc, MaybeOpt::Opt),
+    (Input::PlainText, MaybeEsc::Esc, MaybeOpt::SkipOpt),
+    (Input::PlainText, MaybeEsc::SkipEsc, MaybeOpt::Opt),
+    (Input::PlainText, MaybeEsc::SkipEsc, MaybeOpt::SkipOpt),
```

Comment:
This is the cartesian product of all of the possible input toggles, but any fancy way of expressing that seemed way longer than just listing out the permutations. `args` does accept an iterator, so maybe if we depend on something like `itertools` in the future then we could make this a touch more succinct